docs/sdks/sdk-ref-javascript.md

```diff
@@ -41,9 +41,9 @@ For example applications with associated documentation, see:
 
 This SDK simplifies development for publishers who want to build their own customized UID2 integration. The following table shows the functions it supports.
 
-| Encrypt Raw UID2 to UID2 Token | Decrypt UID2 Token | Generate UID2 Token from DII | Refresh UID2 Token |
-| :--- | :--- | :--- | :--- |
-| Not supported | Not supported | Not supported | Supported |
```

Comment:
Generate UID2 Token from DII used to be "Not supported" - was this a mistake that you are now fixing?

Reply:
yes JS SDK can do CSTG now so should be supporting Generate UID2 Token from DII